docs/usage/package-structure.md

```diff
@@ -0,0 +1,169 @@
+# Important files / folders
+
+## `setup.py`
+The package will follow some conventions, to make it "discoverable" by others.
+
+In `setup.py`, always add the correct Trove Classifiers:
+
+```python
+        "Framework :: Plone",
+        "Framework :: Plone :: 6.0",
+        "Framework :: Plone :: Distribution",
+```
+
+and also require `plone.distribution` to be available:
+
+```python
+    install_requires=[
+        "Plone",
+        "setuptools",
+        "plone.distribution",
+    ],
+```
+
+## `configure.zcml`
+
+In your main `configure.zcml`, make sure to have the `plone` xml namespace declared:
+
+```xml
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:plone="http://namespaces.plone.org/plone"
+    >
+```
+
+And also include `plone.distribution`:
+
+```xml
+  <include package="plone.distribution" />
+```
+
+Then declare the distributions included in your package:
+
+```xml
+
+  <plone:distribution
+      name="blog"
+      title="Personal Blog"
+      description="A Plone site already configured to host a personal Blog."
+      directory="distributions/blog"
+      />
+
+```
+
+The registered distribution will configure a Personal Blog, with some default content.
+
+## `distributions` folder
+
+A convention is to use the `distributions/<distribution_name>`folder in the root of your package to organize your distribution configuration.
+
+In that folder, you will need to provide:
+
+### `image.png`
+
+A 1080x768 image of your distribution. It could be the default page of a new site, your logo, or any other way of representing this distribution.
+
+### `profiles.json`
+
+A `JSON` file with the GenericSetup profiles that are used by your distribution during installation.
+
+This file needs to contain two keys:
+
+* **base**: List of profiles installed in every new site using this distribution.
+
+* **content**: List of profiles installed when the user decides to create a site with example content.
+
+The configuration for a new Volto site is:
+
+```json
+{
+  "base": [
+    "plone.app.contenttypes:default",
+    "plone.app.caching:default",
+    "plonetheme.barceloneta:default",
+    "plone.volto:default"
+  ],
+  "content": [
+    "plone.volto:default-homepage"
+  ]
+}
+```
+
+### `schema.json`
+
+In case you require additional input from the user during site creation, you can customize the form using the `schema.json` file.
+
+The file should contain two keys:
+
+* **schema**: A JSON Schema definition.
+* **uischema**: A [react-jsonschema-form](https://rjsf-team.github.io/react-jsonschema-form/docs/) configuration to modify how the form is displayed.
+
+The **schema** should have at least the following keys:
+
+* site_id
+* title
+* description
+* default_language
+* portal_timezone
+* setup_content
+
+The `schema.json` used for the default site creation is:
```

Comment:
```suggestion
The `schema.json` used for the default site creation is the following.
```

Reply:
To avoid changes in code not getting updated in docs, I wonder if there is a more elegant way of managing them, such as with [`literalinclude`](https://myst-parser.readthedocs.io/en/stable/syntax/code_and_apis.html#including-code-from-files)?